static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -330,7 +330,9 @@ const Segment = styled('span', {shouldForwardProp: isPropValid})<{color: string}
   padding: 1px ${space(0.5)} 0 0;
 `;
 
-const LegendAnimateContainer = styled(motion.div)<{expanded: boolean}>`
+const LegendAnimateContainer = styled(motion.div, {
+  shouldForwardProp: prop => prop !== 'expanded' && isPropValid(prop),
```

Comment:
Hey @scttcper sorry for missing this and thanks for looking into it! I think we might have to explicitly permit the `'animate'` prop (`isPropValid` returns false for some reason) otherwise it actually breaks the `motion.div` functionality. Maybe something like this?
```suggestion
  shouldForwardProp: prop => prop === 'animate' || (prop !== 'expanded' && isPropValid(prop)),
```